pkgs/by-name/pd/pdpmake/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+stdenv.mkDerivation rec {
+  pname = "pdpmake";
+  version = "1.4.1";
+
+  src = fetchFromGitHub {
+    owner = "rmyorston";
+    repo = "pdpmake";
+    rev = version;
+    hash = "sha256-N9MT+3nE8To0ktNTPT9tDHkKRrn4XsTYiTeYdBk9VtI=";
+  };
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  doCheck = true;
+  checkTarget = "test";
+
+  enableParallelBuilding = true;
+
+  meta = with lib; {
```

Comment:
The maintainer and mainprogram tags are missing. Please add them (:

Reply:
Corrected. Would it be better to add `meta.mainProgram` even when `pname` is the same as it?

Reply:
Yes, because relying only on pname is now deprecated as it lead to some bugs earlier on.